test/e2e/cluster_upgrade.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

Comment:
@rmmh From my understanding, BUILD files should only be generated via `hack/update-bazel.sh` without manual modification. I did so, but the compiler still treats `upgrades/apps` an import re-declaration of `upgrades`.

I compared the generated BUILD files with BUILD files of a similar nested directory. There were some discrepancies, but as i should not modify any BUILD file manually, i am not able to do anything regarding this.

Do you mind to elaborate how to import `upgrades/apps` as `apps` _without_ modifying BUILD files?